package/grpc/Config.in

```diff
@@ -0,0 +1,19 @@
+config BR2_PACKAGE_GRPC
+	bool "grpc"
+	depends on BR2_INSTALL_LIBSTDCPP
+	depends on BR2_TOOLCHAIN_HAS_THREADS
+	select BR2_PACKAGE_C_ARES
+	select BR2_PACKAGE_GFLAGS
+	select BR2_PACKAGE_GTEST
+	select BR2_PACKAGE_GTEST_GMOCK
```

Comment:
Do gtest/gflags/gmock always need to be enabled?  Are these installed into the target?

Reply:
Yeah, I think they need to be enabled as far as I can tell. Definitely gflags. But there's no mechanism to to disable tests in the target like there is for protobufs.